core/src/main/java/org/elasticsearch/node/InternalSettingsPreparer.java

```diff
@@ -44,6 +47,9 @@
 
 public class InternalSettingsPreparer {
 
+    private static Logger logger = ESLoggerFactory.getLogger("settings");
```

Comment:
Why name this logger against our convention? It's not even prefixed with "org.elasticsearch" but I think you should only pass `InternalSettingsPreparer.class` here.